### DISCUSSION TITLE
Enhancement: tell the user why their request failed

### DISCUSSION THREAD

**Discussion by jegraham-amplify** [danny-avila/LibreChat#3201] [Feature Requests & Suggestions]
### What features would you like to see added?

UI errors are non-specific and vague. From these messages alone, the user is entirely unable to determine what went wrong and whether it's an issue with the site/server or an issue with their request. When the user has done something incorrect to cause the error, they should be informed.

### More details

Example scenario: uploading an image to a model that doesn't support it

Error shown on the UI: Something went wrong. Here's the specific error message we encountered: An error occurred while processing your request. Please contact the Admin.

Error logged by the server: error: [handleAbortError] AI response error; aborting request: 400 Invalid content type. image_url is only supported by certain models.

Suggested fix: Display more relevant error information to the user

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
They are intentionally non-specific and vague as API errors can leak sensitive information and is not a security best practice.

However there are some errors that are "known" and a custom message is rendered for them specifically. I can add the one you highlighted to the list

> Error logged by the server: error: [handleAbortError] AI response error; aborting request: 400 Invalid content type. image_url is only supported by certain models.

Would be helpful to compile more like this from the community.

**Reply by jegraham-amplify** [reply]
Has this error been added to that list? I just pulled updates and am still not seeing a specific error message propagate to the user when they try using vision with models that don't support it.
